docs/google-cloud.md

```diff
@@ -34,107 +44,121 @@ gcloud auth configure-docker
 PATH="$PATH:$(pwd)/bin"
 ```
 
-### And run the configuration script
+### Run the configuration script
 
 ```
 kctf-config-create
 ```
 
-### Configuration properties
-Type a path for storing the challenges
+### Set configuration properties
+Enter a path for storing the challenges:
 ```
 ~/demo-ctf-cluster
 ```
 
-Type your project id:
+Enter your project id:
 ```
-{{project-id}}
+{{ project-id }}
 ```
 
-You can use the default values for all other settings.
+### Other settings 
+For all other settings, use the default values.
+
+## Create the cluster
+After configuring the project, the cluster is created automatically.
 
-## And finally, create the cluster
-After this is done, the cluster will be created.
+This is only done once per CTF. A "cluster" is essentially a group of VMs with a "master" that defines what to run there.
 
-This only needs to be done once per CTF. A "cluster" is essentially a group of VMs with a "master" that defines what to run there.
+It can take around a minute for the cluster to be created, during which the following message is displayed:
 
-This can take a while, you can expect to see `Creating cluster ... in ... Cluster is being health-checked...` and in a minute or so, you'll get a message telling you it's done.
+```Creating cluster ... in ... Cluster is being health-checked...``` 
+
+You'll get a message when cluster creation has completed.
 
 While you wait, here's how the infrastructure works:
-1. The CTF challenges will run inside "nsjail" (a security sandbox).
```

Comment:
hmm, is this good way to present this? maybe we can instead send the user to the intro doc (the cluster takes so long, that doc is probably more appropriate?).

that said, the goal of this section is to take the opportunity of this "break" in the process to teach the reader something, but maybe we can skip this step if it's too long or doesnt make sense (we can let the cluster be created in the background). wdyt?

Reply:
I agree this is kind of a weird "step", as the user does nothing but wait for an automatic process to complete. We could just add a sentence to the end of the "Configure the project" section saying "After configuring the project, the cluster is created automatically."

On the other hand: By having this separate step, we give the cluster creation more prominence. Maybe a way to decide is: if the info we currently provide in this section is really important for users to know, let's keep it. We could put it somewhere else and link to it, but I doubt many people will follow the link and moving to a new page is always harder than reading a bit of info integrated directly where you are. Thoughts?

Reply:
"we can skip this step if it's too long or doesnt make sense (we can let the cluster be created in the background)" what do you mean by this? This would be a change in the way the set up works?

Reply:
I think the info we provide here is important for the user to know, yes. Mainly because it's likely later on they'll need to know this to be able to fix things. Let's keep it then :)